pkg/didcomm/common/transport/envelope.go

```diff
@@ -6,9 +6,14 @@ SPDX-License-Identifier: Apache-2.0
 
 package transport
 
-// Envelope contain msg, FromVerKey and ToVerKeys
+// Envelope holds message data and metadata for inbound and outbound messaging
 type Envelope struct {
 	Message    []byte
-	FromVerKey string
-	ToVerKeys  []string
+	FromVerKey []byte
+	// ToVerKeys stores string (base58) verification keys for an outbound message
```

Comment:
It's probably unrelated to this issue, but should better understand the selection of base58.

Reply:
Bitcoin-format base58 is our default way to encode keys as strings in the codebase. Moving to a key handle format would allow us to have a `ToString()` method and FromString constructor, for example, so all conversions back and forth are consistent.

Except when they aren't because sometimes we need to convert to Base64, but those can be the exception.

Reply:
Created #1007 for this.